sonar-scanner-engine/src/main/java/org/sonar/scanner/bootstrap/ScannerWsClientProvider.java

```diff
@@ -31,15 +31,15 @@
 import static org.apache.commons.lang.StringUtils.defaultIfBlank;
 
 @ScannerSide
-public class BatchWsClientProvider extends ProviderAdapter {
+public class ScannerWsClientProvider extends ProviderAdapter {
 
   static final int CONNECT_TIMEOUT_MS = 5_000;
   static final String READ_TIMEOUT_SEC_PROPERTY = "sonar.ws.timeout";
   static final int DEFAULT_READ_TIMEOUT_SEC = 60;
 
-  private BatchWsClient wsClient;
+  private ScannerWsClient wsClient;
 
-  public synchronized BatchWsClient provide(final GlobalProperties settings, final EnvironmentInformation env) {
+  public synchronized ScannerWsClient provide(final GlobalProperties settings, final EnvironmentInformation env, GlobalMode globalMode) {
     if (wsClient == null) {
       String url = defaultIfBlank(settings.property("sonar.host.url"), CoreProperties.SERVER_BASE_URL_DEFAULT_VALUE);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "SERVER_BASE_URL_DEFAULT_VALUE"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)